### DISCUSSION TITLE
Admin APIs for project (list/patch)

### DISCUSSION THREAD

**Discussion by Huong-nt** [ory/kratos#2897] [Q&A] [answered]
I try to use admin APIs to list and update my project, however, I always got error `404: Requested url does not match any rules`

Other APIs such as list, get identity still working well

I am doing something wrong?

<img width="1124" alt="Screen Shot 2022-11-16 at 17 21 13" src="https://user-images.githubusercontent.com/10755305/202126979-5a1281ae-a9da-490e-84d3-15cfa58ec961.png">

**Comment by aeneasr** [comment, marked as answer]
Hi :) Unfortunately, using the API of our management plane (so project management) is currently not working easily because we have to yet define a security model. You can however use the Ory CLI to manage the project using a bit of code.

We have it on our roadmap for next year to expose the project management. APIs as regular REST apis.
